changelog_unreleased/javascript/13677-2.md

```diff
@@ -0,0 +1,18 @@
+#### Fix Idempotence for `break|continue|debugger` (#13677 by @liuxingbaoyu)
```

Comment:
Hi @liuxingbaoyu the dangling comment issue fixed by #15826, would you send this fix separately?

Reply:
I tried it and the fix is no longer correct, it doesn't match the behavior of the main branch, so I won't open the new PR anymore.

Reply:
Okay, thank you anyway.